common/include/OpenSpaceNetArgs.h

```diff
@@ -0,0 +1,98 @@
+/********************************************************************************
+* Copyright 2017 DigitalGlobe, Inc.
+* Author: Joe White
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+********************************************************************************/
+
+#ifndef OPENSPACENET_OPENSPACENETARGS_H
+#define OPENSPACENET_OPENSPACENETARGS_H
+
+#include <vector/Feature.h>
+
+#define OSN_LOG(sev) DG_LOG(OpenSpaceNet, sev)
+#define MAPSAPI_MAPID  "digitalglobe.nal0g75k"
+
+
+namespace dg { namespace osn {
+
+enum class Source
+{
+    UNKNOWN,
+    LOCAL,
+    DGCS,
+    EVWHS,
+    MAPS_API,
+    TILE_JSON
+};
+
+enum class Action
+{
+    UNKNOWN,
+    HELP,
+    DETECT,
+    LANDCOVER
+};
+
+class OpenSpaceNetArgs
```

Comment:
From @adpowel (moving from #195):
>Minor point but it seems like this would be better defined as a struct rather than a class since every member is public.